ush/forecast_postdet.sh

```diff
@@ -436,16 +431,7 @@ WW3_postdet() {
     first_ww3_restart_out=$(date --utc -d "${first_ww3_restart_out:0:8} ${first_ww3_restart_out:8:2} + ${half_window} hours" +%Y%m%d%H)
   fi
 
-  # Link restart files
-  for (( vdate = first_ww3_restart_out; vdate <= forecast_end_cycle;
-         vdate = $(date --utc -d "${vdate:0:8} ${vdate:8:2} + ${restart_interval} hours" +%Y%m%d%H) )); do
-    seconds=$(to_seconds "${vdate:8:2}0000")  # convert HHMMSS to seconds
-    ww3_restart_ufs_file="ufs.cpld.ww3.r.${vdate:0:4}-${vdate:4:2}-${vdate:6:2}-${seconds}.nc"
-    ww3_netcdf_restart_file="${vdate:0:8}.${vdate:8:2}0000.restart.ww3.nc"
-    ${NLN} "${DATArestart}/WW3_RESTART/${ww3_netcdf_restart_file}" "${ww3_restart_ufs_file}"
```

Comment:
This portion of the code changed file formats from how the model is writing it out to the more readable YYYYMMDD.HH0000.restart.ww3.nc file name structure.  I think we might need additional changes if this is removed, but maybe I'm not thinking through everything?  

Do you have output of the extended S2SW test on WCOSS2 and/or the low res gfs cycled test so I can take a look at the output/logs or do I need to run these tests?

Reply:
I went with keeping the names as is, just added some clarifying comments

Reply:
Thanks for your checks on the logs.
I will resolve this conversation and open subsequent PRs for the other required work.